subliminal/subtitle.py

```diff
@@ -232,7 +233,9 @@ def guess_matches(video, guess, partial=False):
     if video.resolution and 'screen_size' in guess and guess['screen_size'] == video.resolution:
         matches.add('resolution')
     # format
-    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
+    if video.format and guess.get('format') \
+            and isinstance(guess['format'], str if sys.version_info[0] >= 3 else basestring) \
```

Comment:
@Deathspike 

________________________________ pyflakes-check ________________________________
/home/travis/build/Diaoul/subliminal/subliminal/subtitle.py:237: UndefinedName
undefined name 'basestring'
============== 1 failed, 403 passed, 17 skipped in 323.29 seconds ==============

Reply:
Just check for `list` instead or stick with `str`.